src/components/chart/chart.scss

```diff
@@ -1,7 +1,23 @@
 .chart {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  height: 100%;
   padding: 20px;
 
   &__visual {
     margin: 0 auto;
   }
+
+  &__block {
+    &--hidden {
+      display: none !important;
```

Comment:
why do you need important here?

Reply:
because canvas with chart has inline style `display: block`
![image](https://user-images.githubusercontent.com/14272396/38051554-58928258-32d7-11e8-93a0-1edcc44ae728.png)

Reply:
ok